bin/apikana

```diff
@@ -134,16 +134,17 @@ function generate() {
         var generator = plop.getGenerator('start');
         const model = Object.assign({}, packageJSON, { api: openapi });
 
-        // Workaround for validatorjs to fix wildcard on `paths`
-        const paths = [];
-        Object.values(model.api.paths).forEach(path => paths.push(path));
-        model.api.paths = paths;
-
         if (defaults.validation) {
+
+            // Workaround for validatorjs to validate also 'paths' with wildcard 
+            var openapiToValidate = JSON.parse(JSON.stringify(openapi));
+            openapiToValidate.paths = Object.values(openapiToValidate.paths);
+
             const Validator = require('validatorjs')
-            const validator = new Validator(model.api, defaults.validation);
+            const validator = new Validator(openapiToValidate, defaults.validation);
             if(!validator.check()) {
-                log(colors.bold(colors.red('Validation of the OpenAPI definition has failed with ' + validator.errorCount + ' errors')));
+                log(colors.bold(colors.red('Validation of the OpenAPI (api.yaml) has failed with ' + validator.errorCount + ' error(s).')));
+                log(colors.bold(colors.red('Validation rules your API must conform with are defined in: '+defaults.dir+'\\index.js')));
```

Comment:
Please use the OS path separator instead of '\' as users may develop on a Linux or macOS machine. Here's an idea on how to implement it: https://stackoverflow.com/a/125825

Reply:
thanks for pointing out, its fixed with the last push.